Practica2/includes/vistas/helpers/anuncios.php

```diff
@@ -0,0 +1,33 @@
+<?php
+require_once 'includes/config.php';
+//require_once 'includes/funciones.php'; // Asumiendo que aquí tienes funciones para obtener los anuncios
+
+$tituloPagina = 'Tablón de Anuncios';
+$anuncios = obtenerAnuncios(); // Supongamos que esta función devuelve todos los anuncios en un array
```

Comment:
Poner código que no funciona / existe, complica la corrección / evaluación. O bien sólo generais HTML para tener interfaces cableaas o bien tenéis el código PHP completo y funcional